include/queue/queue.h

```diff
@@ -1,10 +1,13 @@
+#ifndef QUEUE_H
+#define QUEUE_H
+
 /*
     AUTHORS: J. W. Sheridan, Siddharth Mahendraker, Shimi Smith
 */
 
 #include <stdint.h>
 
-#define MAX_QUEUE_SIZE 10
+#define MAX_QUEUE_SIZE 3
```

Comment:
Was this change intentional? Making the MAX_QUEUE_SIZE 3 could be problematic down the road, when we realize we need more room.